lib/WasmReader/WasmByteCodeGenerator.h

```diff
@@ -54,6 +54,7 @@ namespace Wasm
         }
         Js::FunctionBody * body;
         WasmFunctionInfo * wasmInfo;
+        bool imported;
```

Comment:
this is redundant, didn't realize when I added that it was already in WasmFunctionInfo. Will remove this and only use one in WasmFunctionInfo